clients/unshaded/src/main/java/tachyon/client/file/FileInStream.java

```diff
@@ -68,6 +68,9 @@
   private final FileSystemContext mContext;
   /** File information */
   private final FileInfo mFileInfo;
+  /** Constant error message for block ID not cached */
+  private static final String BLOCK_ID_NOT_CACHED = "The block with ID {}"
+      + " could not be cached into Tachyon storage. {}";
```

Comment:
Leave out the `{}` and pass the exception into the log message directly instead of `ioe.getMessage()`. That way we can see the stack trace. See http://alvinalexander.com/blog/post/java/how-print-exception-stack-trace-using-log4j-commons